src/app.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import cookieParser from 'cookie-parser';
+import bodyParser from 'body-parser';
+import { router } from './routes/routes';
+import { queryParser, setCoockies } from './middlewares/middlewares';
+
+const app = express();
+
+app.use(queryParser());
+app.use(cookieParser(), setCoockies());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+app.use('/', router);
```

Comment:
```
// [app.js]
app.use('/api/v1/products', router);
app.use('/api/v1/users');
```
```
// [routers.js]
router.get('/', getAllProducts);
router.get('/:id', getProductById);
router.get('/:id/reviews', getSingleProductReviews);
router.post('/', addNewProduct);
```

There is can define base string via routing.
Use API's version is best practice.